go.mod

```diff
@@ -10,12 +10,12 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/robfig/cron v1.2.0
 	github.com/stretchr/testify v1.8.4
-	go.temporal.io/api v1.26.1
+	go.temporal.io/api v1.26.1-0.20240103185939-608bdd111e4b
```

Comment:
How can we make sure we don't release the SDK with an untagged API version?

Reply:
I'm happy to cut a release of api-go if we're ok with releasing all of https://github.com/temporalio/api-go/compare/v1.26.0...master

Reply:
@cretz we can't cut a release of api-go because we haven't released the gogoproto changes into the wild.

Reply:
Right, I just want to make sure we remember to tag this before next SDK release